### DISCUSSION TITLE
Is there an option for displaying the webpage scale icon as a percentage (100%, 125%, etc)

### DISCUSSION THREAD

**Discussion by jk0893** [Floorp-Projects/Floorp#1173] [Q&A]
I tried searching it in settings and online but couldn't find anything even related. I really miss this QoL feature from Chrome and Firefox, so would be glad if someone pointed me, is it even a thing in Floorp

**Comment by CutterKnife** [comment]
Go to the "Customize Toolbar" page and remove the scaling tool from the Status Bar.
